core/src/findlib/xattr.h

```diff
@@ -86,10 +86,11 @@ struct xattr_parse_data_t {
 /*
  * Internal tracking data.
  */
-struct xattr_data_t {
+struct XattrData {
   POOLMEM* last_fname;
-  uint32_t flags; /* See BXATTR_FLAG_* */
-  uint32_t current_dev;
+  uint32_t flags{0}; /* See BXATTR_FLAG_* */
```

Comment:
Where to find BXATTR_FLAG_ ? Could you make flags a enum class?

Reply:
I thought about it, but refrained from doing it. I was afraid of the side-effects in that `#ifdef` hell. However, I think it can be done without too much work, so I'll look into it.